charts/core/src/Line/Line.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
+
+import { useChartContext } from '../ChartContext';
+
+import { defaultLineOptions } from './config';
+import { LineProps } from './Line.types';
+
+export function Line({ name, data }: LineProps) {
+  const { addChartSeries } = useChartContext();
+
+  useEffect(() => {
+    addChartSeries({
+      ...defaultLineOptions,
+      name,
+      data,
+    });
+  }, [name, data]);
+
+  return null;
+}
+
+Line.displayName = 'Line';
```

Comment:
I didn't implement any unit tests for this component. There's no actual logic or rendering being done so beyond checking implementation details, that seemed correct. I'm very open to other ideas here though since this is probably the pattern we'll see in most of the core components.

Reply:
I feel like testing that `useChartContext` works as expected, and then a solid suite of Chromatic tests will be good for this package